src/jshint.js

```diff
@@ -3592,6 +3604,8 @@ var JSHINT = (function() {
     if (state.tokens.next.value === "extends") {
       advance("extends");
       c.heritage = expression(10);
+    } else {
+      c.heritage = null;
```

Comment:
I don’t know why, but without this some tests (for example, the last new.target test) fail. However, by copying them in a file, JSHint throws the right errors.